WordPress/Classes/Utility/BuildInformation/RemoteFeatureFlag.swift

```diff
@@ -21,6 +21,7 @@ enum RemoteFeatureFlag: Int, CaseIterable {
     case sdkLessGoogleSignIn
     case bloggingPromptsSocial
     case siteEditorMVP
+    case jetpackSocialImprovements
```

Comment:
Thanks for updating the `improvements` bit to this. This clarifies that we're not toggling the Jetpack Social feature as a _whole_, but rather just the improvements. 👍🏼